css/src/mixins/media-queries.scss

```diff
@@ -40,3 +40,14 @@
 		@content;
 	}
 }
+
+/* 
+ * EXCEPTIONS
+ * Please be careful with using these classes. They don't follow the mobile-first approach but might be extremely useful in some use-cases.
```

Comment:
> Please avoid using this mixin if possible. It doesn't follow our the mobile-first approach to writing CSS.

Also, move comment to directly above mixin?

Reply:
Updated and moved right on top of the mixin